components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```
@@ -841,130 +839,55 @@
+    validate_webui_config(clp_config, client_settings_json_path, server_settings_json_path)
 
-    validate_log_viewer_webui_config(clp_config, settings_json_path)
+    # Read, update, and write back client's and server's settings.json
+    client_settings_json_updates = {
+        "ClpStorageEngine": clp_config.package.storage_engine,
+        "MongoDbSearchResultsMetadataCollectionName": clp_config.webui.results_metadata_collection_name,
+    }
+    client_settings_json = read_and_update_settings_json(
+        client_settings_json_path, client_settings_json_updates
+    )
+    with open(client_settings_json_path, "w") as client_settings_json_file:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)